.ci/release.groovy

```diff
@@ -32,6 +32,7 @@ pipeline {
     GCLOUD_VERSION = "320.0.0"
     JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
     CREDENTIALS_FILE = 'credentials.json'
+    IS_USER_TRIGGER = isUserTrigger()
```

Comment:
Where do you use it?

Reply:
https://github.com/elastic/package-storage/blob/dfefbaef9d3b060283fd1d28e6c116e57c109dd4/.ci/release.groovy#L96

Reply:
Aha! I forgot the env variable was created with that step -> https://github.com/elastic/apm-pipeline-library/blob/2617138f96336fd005677ce542f1c6d1d11d35b8/vars/isUserTrigger.groovy#L29-L32

Thanks :)